cryptol-remote-api/python/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for `cryptol` Python package
 
+## 3.2.0 -- 2024-08-20
+
+* The v3.2.0 release is made in tandem with the Cryptol 3.2.0 release. See the
+  Cryptol 3.2.0 release notes for relevant Cryptol changes.
```

Comment:
Make sure to update the version number for the Python bindings in `pyproject.toml` as well.